chart/k8gb/.helmignore

```diff
@@ -20,3 +20,6 @@
 .idea/
 *.tmproj
 .vscode/
+
+LICENSE
+README.md
```

Comment:
@jkremser was late to spot, does this mean that the k8gb helm chart packed and published by next release won't have associated license and readme included in here https://artifacthub.io/packages/helm/k8gb/k8gb ?

Reply:
thanks for catching this, it's fixed by https://github.com/k8gb-io/k8gb/pull/761